mappings/net/minecraft/entity/ai/brain/task/AttackTask.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_4807 net/minecraft/entity/ai/brain/task/AttackTask
+	FIELD field_22285 distance I
+	FIELD field_22286 forwardMovement F
+	METHOD <init> (IF)V
+		ARG 1 distance
+		ARG 2 forwardMovement
+	METHOD method_24551 isAttackTargetVisible (Lnet/minecraft/class_1308;)Z
+		ARG 1 entity
+	METHOD method_24554 isCloseToAttackTarget (Lnet/minecraft/class_1308;)Z
```

Comment:
`isNearAttackTarget` would be shorter. See also https://github.com/FabricMC/yarn/issues/1128.